.gitignore

```diff
@@ -19,3 +19,5 @@ CMakeLists.txt.user
 __pycache__CMakeLists.txt.user
 .gdbinit
 resources
+.pytest_cache
+/expressions.tokens
```

Comment:
why the preceding `/`?

Reply:
Presumably to only ignore that file if its in the top-level directory.

Reply:
Ok. This question was non-blocking anyway, I approve the changes.

Reply:
Yes, I want to defer to the future any decision to ignore this filename everywhere. It's preferable to only ignore it in root for now.